metricflow/test/plan_conversion/dataflow_to_sql/test_conversion_metrics_to_sql.py

```diff
@@ -168,3 +168,32 @@ def test_conversion_rate_with_constant_properties(
         sql_client=sql_client,
         node=dataflow_plan.sink_output_nodes[0].parent_node,
     )
+
+
+@pytest.mark.sql_engine_snapshot
+def test_conversion_metric_fill_nulls_with_0(
+    request: FixtureRequest,
+    mf_test_session_state: MetricFlowTestSessionState,
+    dataflow_plan_builder: DataflowPlanBuilder,
+    dataflow_to_sql_converter: DataflowToSqlQueryPlanConverter,
+    sql_client: SqlClient,
+) -> None:
+    """Test conversion metric with constant properties by data flow plan rendering."""
```

Comment:
Let's update the docstring here and in the other test cases to indicate any weirdness. Like if this won't fill nulls at every level, we can put that in with a note that this should highlight the relevant changes if/when we alter that behavior.